docs/examples/branching_story.py

```diff
@@ -1,11 +1,14 @@
 import columbo
 
+
```

Comment:
This file (and the other updated files in the `docs/examples/` directory) have been either updated or pulled from the documentation.